pages/topics/voice-connections.mdx

```diff
@@ -81,20 +81,35 @@ Having the `MANAGE_CHANNELS` permission bypasses this limit and allows you to jo
 
 </Alert>
 
+<Alert type="info">
+
+Similarly, when the voice channel max video user limit is reached, you will not receive any events in response to your request.
+However, having the `MANAGE_CHANNELS` permission only allows an extra one additional user to join the channel. It does not bypass the limit entirely.
+
+</Alert>
+
+<Alert type="info">
+
+Private channel voice connections work exactly the same as guild voice channels, except that the channel's ID is used as the server ID below.
+If the client is the first to join a private channel, a call will be created for the channel and a [Call Create](/topics/gateway-events#call-create) Gateway event will be fired.
+Once a call exists, clients can choose to [ring the channel recipients](/resources/channel#ring-channel-recipients).
```

Comment:
Confusing, why is this "private channel" and not DM?

Reply:
Private channels encompass DMs and group channels